skiko/src/jvmMain/cpp/common/jni_helpers.cc

```diff
@@ -0,0 +1,110 @@
+#if SK_BUILD_FOR_WIN
+
+#include "jni_helpers.h"
+
+JavaVM *jvm = NULL;
+
+bool isHandleException(JNIEnv *env)
+{
+    static jclass systemClass = NULL;
+    if (!systemClass)
+    {
+        systemClass = env->FindClass("java/lang/System");
+    }
+
+    static jmethodID getPropertyMethod = NULL;
+    if (!getPropertyMethod)
+    {
+        getPropertyMethod = env->GetStaticMethodID(systemClass, "getProperty", "(Ljava/lang/String;)Ljava/lang/String;");
+    }
+
+    jstring propertyName= env->NewStringUTF("skiko.win.renderexceptionhandling.enabled");
+    jstring propertyString = (jstring)env->CallStaticObjectMethod(systemClass, getPropertyMethod, propertyName);
+    if (propertyString == 0)
+    {
+        return false;
+    }
+    const char *property = env->GetStringUTFChars(propertyString, 0);
+    bool result = !strcmp(property, "true");
```

Comment:
`strncmp(property, "true", 4)` would be safer.

Reply:
Done.